boot/zephyr/Kconfig

```diff
@@ -557,7 +557,9 @@ endchoice
 
 config BOOT_WATCHDOG_FEED
 	bool "Feed the watchdog while doing swap"
-	default y
+	default y if WATCHDOG
+	default y if SOC_FAMILY_NRF
```

Comment:
Doesn't `CONFIG_SOC_FAMILY_NRF` allow for disabling `CONFIG_WATCHDOG`? How about a `depends on WATCHDOG`?

Reply:
I do not want to hide the option, I prefer to have it always visible. For nRF the macro-logy goes without watchdog driver (our low-level implementation).